### PR TITLE
punctuation's added

### PR DESCRIPTION
**Pull request by sidharthpunathil**
and ->  ,  and

### CODE REVIEW

**Review by arielshaqed — APPROVED** [review]
Thanks!  We *like* the [Oxford comma](https://en.wikipedia.org/wiki/Serial_comma)...

**Comment by arielshaqed** [comment]
Please resolve conflicts -- there have been others editing the same file -- and then dismiss my review.  I will be happy to merge.

**Review by YaelRiv — APPROVED** [review]
Thanks @sidharthpunathil

**Comment by arielshaqed** [comment]
@sidharthpunathil note that I cannot merge with conflicts; please resolve to unblock me. Thanks!

**Comment by arielshaqed** [comment]
Hi @sidharthpunathil ,
Please note that I am unable to pull until you resolve the conflicts.  There's a button "Resolve conflicts" that might help, or you might prefer to use the command line.
Thanks!

File: README.md [hunk 1]
```diff
@@ -59,16 +59,18 @@ Thanks!
...
 **Continuous Data Integration**
 * **Ingest new data safely by enforcing best practices** - make sure new data sources adhere to your lake’s best practices such as format and schema enforcement, naming convention, etc.  
 * **Metadata validation** - prevent breaking changes from entering the production data environment.
 
 **Continuous Data Deployment**
-* **Instantly revert changes to data** - if low quality data is exposed to your consumers, you can revert instantly to a former, consistent and correct snapshot of your data lake.
-* **Enforce cross collection consistency** - provide to consumers several collections of data that must be synchronized, in one atomic, revertible, action.
+
+* **Instantly revert changes to data** - if low quality data is exposed to your consumers, you can revert instantly to a former, consistent, and correct snapshot of your data lake.
+* **Enforce cross collection consistency** - provide to consumers several collections of data that must be synchronized, in one atomic, revertable, action.
```
Comment (arielshaqed) [inline README.md:72]: Please undo this change "revertible" -> "revertable".  What happened was a concurrent PR fixed this spelling and now your PR is changing it back to the previous incorrect version.

**Review by arielshaqed — COMMENTED** [review]
Thanks!  Nearly there; please remove the rollback of an unrelated change and I can merge.

**Comment by arielshaqed** [comment]
> Thanks! Nearly there; please remove the rollback of an unrelated change and I can merge.

@sidharthpunathil FYI this on the last review round is still blocking me from merging.

**Review by sidharthpunathil — COMMENTED** [review]
revertable
